tests/serializers/test_natural.py

```diff
@@ -67,6 +67,36 @@ def natural_key_test(format, self):
     self.assertIsNone(books[1].object.pk)
 
 
+class NaturalKeyWhenPkHasDefaultValueTestCase(TestCase):
+    """
+    Verify that deserializer does not ignore natural keys when primary key has a default value (ticket `28385`).
+
+    Test to prove that deserializer gets `pk` via `natural key` and updates object instead of create new one.
+    """
+
+    fixtures = ['fixture.json', 'fixture.xml']
+
+    def test_natural_key_json(self):
+        """
+        Test fixture based on `json` format.
+        """
+        management.call_command("loaddata", "fixture.json", verbosity=0)
+        self.assertEqual(Foo.objects.all().count(), 1)
```

Comment:
What I mean by "assert the values of the one object that exists in the db across imports, i.e. the pk didn't change" is this:
```python
load_data()
old_obj = Foo.objects.get()  # This already asserts that there's exactly one object in the table
load_data()
new_object = Foo.objects.get()  # This again asserts a single objects in the table
self.assertEqual(old_obj.pk, new_obj.pk)
```